CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 18.2.2 - [#742](https://github.com/openfisca/openfisca-france/pull/742)
```

Comment:
The CI should not enforce a version bump if you haven't change any file inside `openfisca_france ` ;)